include/AudioUnitSDK/AUInputElement.h

```diff
@@ -58,7 +58,7 @@ inline OSStatus AUInputElement::PullInputWithBufferList(AudioUnitRenderActionFla
 	const AudioTimeStamp& inTimeStamp, AudioUnitElement inElement, UInt32 nFrames,
 	AudioBufferList& inBufferList)
 {
-	OSStatus theResult = noErr;
+	OSStatus theResult;
```

Comment:
There are a bunch of changes in this commit that look like this, and there is no good reason for them and no way that we are going to adopt these. We have a policy in the project to not leave variables uninitialized.